charge/clients/logging.py

```diff
@@ -0,0 +1,78 @@
+from autogen_ext.models.openai import OpenAIChatCompletionClient
+
+from openai.types.chat import ChatCompletion
+from loguru import logger
+
+## Create a custom wrapper to log responses
+class LoggingModelClient(OpenAIChatCompletionClient):
```

Comment:
A custom client just to log responses seems to be overkill. We should be able to do this through the context object

Reply:
Well maybe, but this seems to be working now.